app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if !logged_in?
+      redirect to '/login'
+    else
+      @tweets = Tweet.all
+      erb :'/tweets/index'
+    end
+  end
+
+  get '/tweets/new' do
+    if !logged_in?
+      redirect to '/login'
+    else
+      erb :'/tweets/new'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to 'tweets/new'
+    else
+      @tweet = current_user.tweets.build(params)
```

Comment:
Another way to do this is to make use of the `.create` method provided by the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```

That way you don't have to call `.save` on the `current_user`. :)